Makefile

```diff
@@ -270,3 +285,7 @@ production-cluster:
 get-cluster-credentials: set-azure-account
 	az aks get-credentials --overwrite-existing -g ${CLUSTER_RESOURCE_GROUP_NAME} -n ${CLUSTER_NAME}
 	kubelogin convert-kubeconfig -l $(if ${GITHUB_ACTIONS},spn,azurecli)
+
+dev-cluster:
+	$(eval CLUSTER_RESOURCE_GROUP_NAME=s189d01-tsc-ts-rg)
+	$(eval CLUSTER_NAME=s189d01-tsc-test-aks)
```

Comment:
neither value exist?

Reply:
removed $(eval CLUSTER_NAME=s189d01-tsc-test-aks) whic is not required